src/Bicep.Core/PrettyPrint/PrettyPrintOptions.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using Bicep.Core.PrettyPrint.Options;
+
+namespace Bicep.Core.PrettyPrint
+{
+    public class PrettyPrintOptions
+    {
+        private const int MinIndentSize = 1;
+
+        private const int MaxIndentSize = 8;

```

Comment:
>MaxIndentSize [](start = 26, length = 13)

Can LSP give us higher indent sizes? If yes, why would we limit it in the formatter?

Reply:
It can. On second thought I was overthinking. I thought we should add some constrains in case users set a really big tab size like `100000`, but apparently VSCode accepts that and outputs it upon pressing Tab...although it takes a fews seconds to complete the insertion.
![image](https://user-images.githubusercontent.com/16367959/98174503-d6623a00-1ea9-11eb-85f4-4b815c95291c.png) 

If users decide to be creative, we should support them 💪. I'll remove the limits.

Reply:
Lol, I'm glad you tested that :)

Reply:
Fun fact - after digging deeper I realized that you can overflow the tab size with a huge number: 
![image](https://user-images.githubusercontent.com/16367959/98186461-a0ca4a80-1ec3-11eb-913d-41f40eced671.png)
...which seems to be a bug of VSCode?

Also VSCode does have a minimum tab size of `1`. Now I wonder maybe it's a good idea to keeping both limits, but maybe we can increase the upper boundary a bit to, say, `1000`?

@majastrz @anthony-c-martin what do you think?

Reply:
Makes sense to me.

---
In reply to: [517730586](https://github.com/Azure/bicep/pull/823#discussion_r517730586) [](ancestors = 517730586)